<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsScreen</class>
 <widget class="QWidget" name="SettingsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblAccount">
     <property name="text">
      <string>Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomPlacementButton" name="btnRenew">
     <property name="text">
      <string>Renew subscription</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomPlacementButton" name="btnLogOut">
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblSettings">
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboBox" name="cbbLanguage">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>259</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>ComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CustomPlacementButton</class>
   <extends>QPushButton</extends>
   <header location="global">CustomPlacementButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
